<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabLocations</class>
 <widget class="QWidget" name="TabLocations">
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Number of samples:</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="5">
    <widget class="QLabel" name="labelStatus">
     <property name="font">
      <font>
       <family>Monospace</family>
      </font>
     </property>
     <property name="text">
      <string>Idle</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="4">
    <widget class="QLineEdit" name="lineN">
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <widget class="QLineEdit" name="lineZ"/>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;X:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Sampling strategy:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="5">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushExport">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Export...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushExpand">
       <property name="text">
        <string>Expand all</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushStart">
       <property name="text">
        <string>Analyze</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1" colspan="4">
    <widget class="QComboBox" name="comboSampling">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="5">
    <widget class="QTreeWidget" name="treeWidget">
     <property name="font">
      <font>
       <family>Monospace</family>
      </font>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::NoDragDrop</enum>
     </property>
     <property name="indentation">
      <number>12</number>
     </property>
     <property name="animated">
      <bool>true</bool>
     </property>
     <property name="expandsOnDoubleClick">
      <bool>false</bool>
     </property>
     <attribute name="headerDefaultSectionSize">
      <number>100</number>
     </attribute>
     <column>
      <property name="text">
       <string>id</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>seed/condition</string>
      </property>
      <property name="font">
       <font>
        <family>Monospace</family>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>x</string>
      </property>
      <property name="font">
       <font>
        <family>Monospace</family>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>z</string>
      </property>
      <property name="font">
       <font>
        <family>Monospace</family>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>distance</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="2" column="1" colspan="4">
    <widget class="QLineEdit" name="lineA">
     <property name="text">
      <string>512</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLineEdit" name="lineX"/>
   </item>
   <item row="4" column="3">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Z:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Centered on:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Spread (Î±):</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Seed(s):</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="4">
    <widget class="QComboBox" name="comboSeedSource">
     <item>
      <property name="text">
       <string>Current seed</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>From matching seeds list</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
